spec/features/visitors/password_recovery_spec.rb

```diff
@@ -89,8 +89,7 @@ def reset_password_and_sign_back_in
       open_last_email
       click_first_link_in_email
       reset_email
-      visit root_path
-      click_link "Didn't receive confirmation instructions?"
+      visit new_user_confirmation_path
```

Comment:
Will there be an alternative option? How will we be allowing users who didn't receive the confirmation email to request the email sent to them again?